src/pages/gallery/index.tsx

```diff
@@ -545,6 +546,14 @@ export default function Gallery() {
         clearSelection();
     };
 
+    const downloadHelper = async () => {
```

Comment:
Function names should ideally be a verb. Since they "do" something. In this case, it could be `downloadSelectedFiles`.

Reply:
Not resolved?

Reply:
ya....we discussed na.. ki there are many function which same nomencature will resolve it in a seperate PR